includes/class-sensei-utils.php

```diff
@@ -1732,6 +1732,7 @@ public static function update_lesson_status( $user_id, $lesson_id, $status = 'in
 		if ( ! empty( $status ) ) {
 			$args = array(
 				'user_id'   => $user_id,
+				'username'  => get_userdata( $user_id )->user_login ?? null,
```

Comment:
Maybe I'm missing something, just being curious, are we using this value somewhere?

Reply:
Got it 👍 it's for the comment_author check

Reply:
Yes, just using a similar optimization to avoid having to add a join to the usermeta table.